### DISCUSSION TITLE
タグの一覧を取得したい

### DISCUSSION THREAD

**Discussion by kay-adamof** [increments/qiita-discussions#567] [ヘルプ] [answered]
`/api/v2/tags` を使ってタグの総数を調べると、総数 が２０万ちかくとなっています。

![image](https://github.com/increments/qiita-discussions/assets/57440326/1a53e212-4c9c-4fec-9c1c-c7ef2499552c)

どのようなタグが使われているのかを知るために、すべてのtagの情報を取得したいと思っています。

しかし、`/api/v2/tags` で取得できるのは、`page x per_page = 10000 (１万 ）` までだと思うのですが、２０万近いtagの情報を取得することはできないのでしょうか？

<img width="486" alt="image" src="https://github.com/increments/qiita-discussions/assets/57440326/cb6da34f-a83c-4aeb-9a95-59f003bbee62">

**Comment by tanaka-asami415** [comment, marked as answer]
@kay-adamof 
お問い合わせありがとうございます。
大変恐れ入りますが、パフォーマンスの都合上、10000 が上限となっております。ご理解いただけますと幸いです🙇
